crate/client/src/findex_rest_client.rs

```diff
@@ -21,11 +21,13 @@ use crate::{
 #[derive(Clone)]
 pub struct FindexClient {
     pub server_url: String,
-    client: Client,
+    pub client: Client,
```

Comment:
It seems this is just an HTTPS client. I would suggest not to store it in some `FindexClient`. This would make everything simpler to read (this HTTPS client has nothing to do with Findex, and could be used by any other protocol built over HTTPS).

Reply:
`FindexClient` could be renamed at least I suppose